runtime/jvmti/jvmtiGeneral.c

```diff
@@ -188,12 +188,9 @@ jvmtiGetVersionNumber(jvmtiEnv* env,
 	ENSURE_NON_NULL(version_ptr);
 
 #if JAVA_SPEC_VERSION >= 11
-#if JAVA_SPEC_VERSION >= 15
-	if (J2SE_VERSION(vm) >= J2SE_V15) {
+	if (J2SE_VERSION(vm) > J2SE_V11) {
```

Comment:
There's no need to use `J2SE_VERSION` at all here.

Reply:
Updated.